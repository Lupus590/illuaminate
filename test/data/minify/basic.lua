for i = 1, 10 do
  print("Hello", i * 2)
end
