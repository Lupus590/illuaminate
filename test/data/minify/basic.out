local function e(t,...)print(string.format(t,...))end local a,o={},1for
i=1,10do
e("Hello",i*2)end